client/store/weatherStore.js

```diff
@@ -16,6 +16,8 @@ const weatherAdapter = createEntityAdapter();
 const initialState = weatherAdapter.getInitialState({
```

Comment:
We are trying to move away from redux for these types of requests. So why are we adding things into here ?